Condition expressions/Single-line if-else statement/task.py

```diff
@@ -0,0 +1,6 @@
+from random import random  # Importing a pseudo-random number generator module
+
+my_random_number = random() * 100
+
+print(my_random_number) if my_random_number > 50 else print("Too small!")
+
```

Comment:
I have two empty lines here in the educator mode for some reason and none in the student mode.
I am not sure if it is some Win magic. @sofiiako , if it is OK in your setup I'll merge